src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3610,12 +3768,41 @@ private SyntaxToken TryEatCheckedOrHandleUnchecked(ref SyntaxToken operatorKeywo
             return TryEatToken(SyntaxKind.CheckedKeyword);
         }
 
-        private OperatorDeclarationSyntax ParseOperatorDeclaration(
+        private MemberDeclarationSyntax ParseOperatorDeclaration(
             SyntaxList<AttributeListSyntax> attributes,
             SyntaxListBuilder modifiers,
             TypeSyntax type,
             ExplicitInterfaceSpecifierSyntax explicitInterfaceOpt)
         {
+            // We can get here after seeing `explicit` or `implicit` or `operator`.  `ret-type explicit op ...` is not
```

Comment:
the change to pull misplaced members into the prior type was adding some additional errors in some "off the rqails" error recovery cases involving invalid operators.  Specifically an operator written like `int explict operator (...`  

This was because the caller first parses out the return type, then actually attempt to parse this as a *non-conversion* operator, which goes COMPLETELY OFF HTE RAILS.  

This makes that scenario much more pleasant, with great error messages,and much nicer trees.